### PR TITLE
Remove APPCALL

### PR DESCRIPTION
**Pull request by erikzhang**
This PR removes the `APPCALL` and other instructions for the same function. Then I will create a new SYSCALL - `System.Contract.Call` as a replacement. In this way we can reduce the coupling to some extent.

### CODE REVIEW

File: src/neo-vm/OpCode.cs [hunk 1]
```diff
@@ -745,22 +737,6 @@ public enum OpCode : byte
         /// Current context is copied to the invocation stack (similar to CALL). Read return count (1-byte), parameter count (1-byte) and a 2-byte value n for jump to be performed.
         /// </summary>
         CALL_I = 0xE0,
```
Comment (shargon) [inline src/neo-vm/OpCode.cs]: `CALL` ?

Comment (erikzhang) [reply src/neo-vm/OpCode.cs]: I will remove this opcode in next PR.

**Comment by erikzhang** [comment]
> This SYSCALL will get `rvcount` and `pcount` from the stack, right?

I am thinking about two ways:

1. Get `rvcount` and `pcount` from the stack.
2. The number of parameters and return values for all contracts are fixed: object main(string operation, object[] args)

Which way do you think that is better?

**Comment by shargon** [comment]
I prefer the option 1, is more flexible

**Comment by erikzhang** [comment]
Another option is to read `rvcount` and `pcount` from the contract's manifest file.

**Comment by shargon** [comment]
Maybe this option is better

**Pull request merged**